### ISSUE TITLE
Will it be migrated to Mantine v7?

### ISSUE THREAD

**Issue by SukkaW** [mantinedev/ui.mantine.dev#64]
As the title.

Most of the existing demos are relying on the `sx` prop and the `createStyles` API. I am wondering if all demos are compatible with Mantine v7, is CSS Modules + CSS Variable powerful enough to implement those demos?

**Comment by rtivital** [comment]
Yes, Mantine UI will be migrated to 7.0. New styles api in 7.0 provides more opportunities to customize components styles, not less.

**Comment by sowizz** [comment]
Is it possible to access the version built with v6?

**Comment by rtivital** [comment]
https://v6-ui-mantine.netlify.app/
